packages/site/cms-site/src/blocks/PixelImageBlock.tsx

```diff
@@ -67,34 +52,34 @@ export const PixelImageBlock = withPreview(
             );
         }
 
-        if (disableBlurPlaceholder) {
+        const blurDataUrl = createDominantImageDataUrl(dimensions.width, dimensions.height, damFile.image.dominantColor);
+
+        if (fill) {
             return (
                 <NextImage
                     loader={(loaderProps) => generateImageUrl(loaderProps, usedAspectRatio)}
                     src={urlTemplate}
-                    width={dimensions.width}
-                    height={dimensions.height}
-                    layout={layout}
-                    priority
+                    fill
+                    placeholder="blur"
+                    blurDataURL={blurDataUrl}
                     alt={damFile.altText ?? ""}
                     {...nextImageProps}
                 />
             );
         }
 
-        const blurDataUrl = createDominantImageDataUrl(dimensions.width, dimensions.height, damFile.image.dominantColor);
         return (
-            <NextImage
-                loader={(loaderProps) => generateImageUrl(loaderProps, usedAspectRatio)}
-                src={urlTemplate}
-                width={dimensions.width}
-                height={dimensions.height}
-                layout={layout}
-                placeholder="blur"
-                blurDataURL={blurDataUrl}
-                alt={damFile.altText ?? ""}
-                {...nextImageProps}
-            />
+            <ImageContainer $aspectRatio={usedAspectRatio}>
+                <NextImage
+                    loader={(loaderProps) => generateImageUrl(loaderProps, usedAspectRatio)}
+                    src={urlTemplate}
+                    fill
```

Comment:
Why do you have the special treatment of `fill` if you add it here as well?

Reply:
as discussed, for a customisation option i offer a version without container, when the fill prop is set